### ISSUE TITLE
Incomplete plural form returned from toPotString

### ISSUE THREAD

**Issue by NervosaX** [connectedcars/gettext-extractor#5]
This is more of an assumption, because I don't have a deep understanding of the pot file structure, however it  seems that some parts of the plural msgstr is omitted. 

Looking into the code, it looks like this is from a third party library. But what I would expect from:

```javascript
gettext("One new message", "{{n}} new messages", 1);
```
is the following:

```po
#: tests/e2e/fixtures/js/view.jsx:22
msgid "One new message"
msgid_plural "{{n}} new messages"
msgstr[0] ""
msgstr[1] ""
```

instead I receive:

```po
#: tests/e2e/fixtures/js/view.jsx:22
msgid "One new message"
msgid_plural "{{n}} new messages"
msgstr[0] ""
```

The change is fairly minor, and applications such as poedit seem to deal with it alright, however when parsing it to json using [i18next-gettext-converter](https://github.com/i18next/i18next-gettext-converter) it does not include the plural form in the output. If it looks like the first example when running through the convert, it does work.

To fix this for now, I run the output generated from gettext-extractor through [pofile](https://github.com/rubenv/pofile) which adds the missing values. This suggests that the extra values should be there (and that does make sense).

pofile has the added benefit of including some headers as well.

**Comment by lukasgeiter** [comment]
Thanks for your feedback! I was never really sure if one of them is more correct than the other, but during research I stumbled upon this rather recent [discussion at the pofile repo](https://github.com/rubenv/pofile/pull/24) and it seems like there should be an empty entry for every plural version...

Unfortunately this is not something I can easily change. I'm using another package to generate the .pot file [gettext-parser](https://github.com/smhg/gettext-parser) which behaves this way. I'm considering switching to pofile for this part, but would like to hold off on it for a while because it's a pretty major change.

**Comment by lukasgeiter** [comment]
This is fixed with version 3.0.0
